docs/operations_tasks/Retry-Failed-Bundle-Exporting.md

```diff
@@ -7,62 +7,22 @@ parent: Operations tasks
 # How to retry failed bundles in the Ingest Exporter
```

Comment:
What does "bundle" mean in this context?

Reply:
Good question - wasn't my writing. I suppose "submissions"

Reply:
A consistent set of terms would help readers and maintainers of this doc.

Reply:
I think bundles makes sense here actually. Exporting is done on a per experiment basis not a submission basis so it's not really the submission envelope.

Reply:
no one speaks of the b* word now, let's just use "experiment" :)))

Reply:
Done!